perception/traffic_light_recognition/traffic_light_map_based_detector/src/node.cpp

```diff
@@ -375,15 +399,15 @@ void MapBasedDetector::publishVisibleTrafficLights(
     point.z = tf_camera2tl.getOrigin().z();
     marker.points.push_back(point);
 
-    marker.lifetime = ros::Duration(0.2);
+    marker.lifetime = rclcpp::Duration(0.2);
```

Comment:
I think this is implicitly calling constructor with nanosec.
You can use `rclcppp::Duration::from_seconds(0.2)` function instead.

Reply:
Thanks, done.